app/app.go

```diff
@@ -27,6 +29,17 @@ var db databases.Database
 // log is a package global used for logging
 var log *logrus.Logger
 
+// This part is just for Errors
+var (
+	unableConnectDB = "Unable to connect to Database"
+	unableInitDB    = "Unable initilize the Database"
+)
+
+// This part is for constant
+var (
+	cassandraName = "casandra"
```

Comment:
Do we really need this when we only use it once?

Reply:
not really , but if the program grows then might be easier

Reply:
i removed it , we can added later if we had to